display meta data test fail

__Describe the Bug__

Display meta data test failed:

 `<ElementTemplatesView>                                                                          
    basics                                                                                        
      ✓ should render                                                                             
      ✓ should get element templates on mount                                                     
      ✓ should get element templates for selected element type                                                                                                                                      
      ✓ should get element templates sorted alphabetically                                                                                                                                          
      ✓ should list element templates                                                             
      ✗ should display meta data                                                                  
        AssertionError: expected 'Walt\'s Catalog | 2001-09-08' to equal 'Walt\'s Catalog | 2001-09-09'                                                                                             
            at Context.eval (webpack:///./src/plugins/element-templates-modal/components/__tests__/ElementTemplatesViewSpec.js?:91:30)`

Here full log:

0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   '/usr/bin/node',
1 verbose cli   '/usr/share/npm/bin/npm-cli.js',
1 verbose cli   'run',
1 verbose cli   'client:test'
1 verbose cli ]
2 info using npm@6.14.8
3 info using node@v12.18.4
4 verbose run-script [ 'preclient:test', 'client:test', 'postclient:test' ]
5 info lifecycle camunda-modeler-builder@0.0.0~preclient:test: camunda-modeler-builder@0.0.0
6 info lifecycle camunda-modeler-builder@0.0.0~client:test: camunda-modeler-builder@0.0.0
7 verbose lifecycle camunda-modeler-builder@0.0.0~client:test: unsafe-perm in lifecycle true
8 verbose lifecycle camunda-modeler-builder@0.0.0~client:test: PATH: /usr/share/npm/node_modules/npm-lifecycle/node-gyp-bin:/home/marciotex/devel/camunda-modeler/node_modules/.bin:/usr/share/npm/node_modules/npm-lifecycle/node-gyp-bin:/home/marciotex/devel/camunda-modeler/node_modules/.bin:/home/marciotex/bin:/usr/local/bin:/usr/bin:/bin:/usr/local/games:/usr/games
9 verbose lifecycle camunda-modeler-builder@0.0.0~client:test: CWD: /home/marciotex/devel/camunda-modeler
10 silly lifecycle camunda-modeler-builder@0.0.0~client:test: Args: [ '-c', '(cd client && npm run test)' ]
11 silly lifecycle camunda-modeler-builder@0.0.0~client:test: Returned: code: 1  signal: null
12 info lifecycle camunda-modeler-builder@0.0.0~client:test: Failed to exec client:test script
13 verbose stack Error: camunda-modeler-builder@0.0.0 client:test: `(cd client && npm run test)`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (/usr/share/npm/node_modules/npm-lifecycle/index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:315:20)
13 verbose stack     at ChildProcess.<anonymous> (/usr/share/npm/node_modules/npm-lifecycle/lib/spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:315:20)
13 verbose stack     at maybeClose (internal/child_process.js:1021:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)
14 verbose pkgid camunda-modeler-builder@0.0.0
15 verbose cwd /home/marciotex/devel/camunda-modeler
16 verbose Linux 5.8.0-2-amd64
17 verbose argv "/usr/bin/node" "/usr/share/npm/bin/npm-cli.js" "run" "client:test"
18 verbose node v12.18.4
19 verbose npm  v6.14.8
20 error code ELIFECYCLE
21 error errno 1
22 error camunda-modeler-builder@0.0.0 client:test: `(cd client && npm run test)`
22 error Exit status 1
23 error Failed at the camunda-modeler-builder@0.0.0 client:test script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]


Sorry, I dont know formatting better.

__Steps to Reproduce__

<!-- Clear steps that allow us to reproduce the error. -->

1. git pull
2. npm install
3. npm test

And then the error above.

__Expected Behavior__

should display meta data test run sucessfull

__Environment__

- OS: Debian GNU/Linux sid
- node --version: v12.18.4

Regards,

--- a/client/src/plugins/element-templates-modal/components/ElementTemplatesModalView.js
+++ b/client/src/plugins/element-templates-modal/components/ElementTemplatesModalView.js
@@ -414,7 +414,7 @@ function leftPad(string, length, character) {
   return string;
 }
 
-function getDate(elementTemplate) {
+export function getDate(elementTemplate) {
   const { metadata } = elementTemplate;
 
   if (!metadata) {

--- a/client/src/plugins/element-templates-modal/components/__tests__/ElementTemplatesModalViewSpec.js
+++ b/client/src/plugins/element-templates-modal/components/__tests__/ElementTemplatesModalViewSpec.js
@@ -14,13 +14,19 @@ import React from 'react';
 
 import { mount } from 'enzyme';
 
-import ElementTemplatesView, { ElementTemplatesListItem, ElementTemplatesListItemEmpty } from '../ElementTemplatesModalView';
+import ElementTemplatesView, {
+  ElementTemplatesListItem,
+  ElementTemplatesListItemEmpty,
+  getDate
+} from '../ElementTemplatesModalView';
 import Dropdown from '../Dropdown';
 
 import BpmnModdle from 'bpmn-moddle';
 import camundaModdlePackage from 'camunda-bpmn-moddle/resources/camunda';
 
 const moddle = new BpmnModdle({ camunda: camundaModdlePackage });
+
+const timestamp = 1000000000000;
 
 
 describe('<ElementTemplatesView>', function() {
@@ -118,7 +124,7 @@ describe('<ElementTemplatesView>', function() {
 
       const meta = listItem.find('.element-templates-list__item-meta').first();
 
-      expect(meta.text()).to.equal('Walt\'s Catalog | 2001-09-09');
+      expect(meta.text()).to.equal(`Walt's Catalog | ${ getDate(new Date(timestamp)) }`);
     });
 
 
@@ -361,11 +367,11 @@ const DEFAULT_ELEMENT_TEMPLATES = [
     metadata: {
       catalogOrganizationId: '00000000-0000-0000-0000-000000000000',
       catalogTemplateId: '00000000-0000-0000-0000-000000000001',
-      created: 1000000000000,
+      created: timestamp,
       tags: [
         'Donald\'s Catalog'
       ],
-      updated: 1000000000000
+      updated: timestamp
     },
     name: 'Template 2',
     properties: []
@@ -378,11 +384,11 @@ const DEFAULT_ELEMENT_TEMPLATES = [
     metadata: {
       catalogOrganizationId: '00000000-0000-0000-0000-000000000000',
       catalogTemplateId: '00000000-0000-0000-0000-000000000002',
-      created: 1000000000000,
+      created: timestamp,
       tags: [
         'Donald\'s Catalog'
       ],
-      updated: 1000000000000
+      updated: timestamp
     },
     name: 'Template 3',
     properties: []
@@ -396,11 +402,11 @@ const DEFAULT_ELEMENT_TEMPLATES = [
     metadata: {
       catalogOrganizationId: '00000000-0000-0000-0000-000000000000',
       catalogTemplateId: '00000000-0000-0000-0000-000000000000',
-      created: 1000000000000,
+      created: timestamp,
       tags: [
         'Walt\'s Catalog'
       ],
-      updated: 1000000000000
+      updated: timestamp
     },
     name: 'Template 1',
     properties: []

